WordPress/src/main/java/org/wordpress/android/util/image/ImageManager.kt

```diff
@@ -79,55 +78,63 @@ class ImageManager @Inject constructor() {
         }
     }
 
+    private fun addPlaceholder(request: GlideRequest<Drawable>, imageType: ImageType): GlideRequest<Drawable> {
```

Comment:
this might be a good usecase for a private extension function, especially since this is basically a builder pattern

Reply:
Great idea, it looks better now!